libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -329,6 +329,9 @@ class AP_AHRS {
     // returns true on success and results are placed in innovations and variances arguments
     bool get_vel_innovations_and_variances_for_source(uint8_t source, Vector3f &innovations, Vector3f &variances) const WARN_IF_UNUSED;
 
+    //returns index of active source set used
+    uint8_t get_active_source_set();
```

Comment:
Could we rename this to, "get_posvelyaw_source_set" and move it up under, "set_posvelyaw_source_set"?  Sorry for not spotting this sooner.

Reply:
Hi, that makes sense and aligns with the current naming, I pushed the changes mentioned
thanks